sdk/postgresql/Azure.ResourceManager.PostgreSql/src/PostgreSqlFlexibleServers/Custom/Models/PostgreSqlFlexibleServerCapabilityProperties.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.PostgreSql.FlexibleServers.Models
+{
+    /// <summary> Capability for the PostgreSQL server. </summary>
+    public partial class PostgreSqlFlexibleServerCapabilityProperties : CapabilityBase
+    {
+        /// <summary> List of supported flexible server editions. </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public IReadOnlyList<PostgreSqlFlexibleServerEditionCapability> SupportedFlexibleServerEditions
+        {
+            get => SupportedServerEditions;
+        }
```

Comment:
So to confirm, these properties adding back in this partial class, are the properties that disappeared in the new version?

Reply:
Yes new version doesn't need any of these properties. They were added back to resolve API compat errors